ci/upload_to_owncloud.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+###############################################################
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# Copyright (c) 2020 Tomer Eliyahu (Intel)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+###############################################################
+
+scriptdir="$(cd "${0%/*}" && pwd)"
+rootdir=$(realpath "$scriptdir/../")
+
+# shellcheck source=functions.sh
```

Comment:
Does this work? I'd expect it should be
```suggestion
# shellcheck source=../tools/functions.sh
```

Reply:
It doesn't work either way, but I'll change to your suggestion